pkg/operation/botanist/worker.go

```diff
@@ -80,6 +80,9 @@ func (b *Botanist) DeployWorker(ctx context.Context) error {
 			UserData: []byte(b.Shoot.OperatingSystemConfigsMap[worker.Name].Downloader.Data.Content),
 			Volume:   volume,
 			Zones:    worker.Zones,
+			ProviderConfig: &runtime.RawExtension{
+				Raw: worker.ProviderConfig.Raw,
```

Comment:
Can you please add a nil check for `worker.ProviderConfig` before adding it?

Reply:
Yes, sorry about that